FARM CONTRACT ADDED TO EXISTING CODE

hulkakshay opened the pull request:
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):


##### Twitter Link:


##### List of audit links if any:


##### Website Link:


##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):


##### Current TVL:


##### Chain:


##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)


##### Short Description (to be shown on DefiLlama):


##### Token address and ticker if any:


##### Category (full list at https://defillama.com/categories) *Please choose only one:


##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):


##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):

g1nt0ki commented:
@hulkakshay can you remove the web3 dependency?

vrtnd commented on package.json:
```diff
@@ -37,7 +37,8 @@
     "tron-format-address": "^0.1.8",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "web3": "^1.7.5"
```
No need to install web3, u can use defillama/sdk instead

vrtnd replied:
Ah, nvm already mentioned

hulkakshay replied:
ok i will try to do that change and will create new PR, aslo one more doubt do i need write all the info about our token again while submitting PR?, because its already listed and i am just adding one more contract.

hulkakshay replied:
The changes are done and web3 is removed,please kindly check. @vrtnd

hulkakshay commented:
ok i will try to remove the web3 dependency and push the code again

hulkakshay commented:
> @hulkakshay can you remove the web3 dependency?

hi i have updated the code and removed web3 dependency , can you please check and merge the branch.

llamatester commented:
Error while running adapter at projects/prxy: 
        
 
  
    
    
    Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:223:11

hulkakshay commented:
> g1nt0ki



> Error while running adapter at projects/prxy:
> 
> ```
> Error: Protocol doesn't have total tvl, make sure to export a tvl key either on the main object or in one of the chains
> Truncated error stack:
> 
>     at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:223:11
> ```

I have removed the extra code that woud cause the total TVL error, and just farm contract is added. I guess it should perfectly now.

hulkakshay commented:
>
